### PR TITLE
semantic search support 

### PR DESCRIPTION
**Pull request by ruoccofabrizio**
## Purpose

* ...
* Added semantic query support
* Added answer support for semantic query
* Added toggle for semantic support
* Added scroll bar for facets
* Updated SDK to 11.4.0-beta for new semantic config

## Does this introduce a breaking change?

```
[ ] Yes
[X ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Review by farzad528 — APPROVED** [review]

**Comment by dereklegenzoff** [comment]
This looks great! Thanks for putting it together. I left a couple comments. Only other thought--would it be possible to add a couple lines in one of the READMEs talking about how to use the semantic search functionality? It could probably just be a couple sentences. For example, the index creation steps don't create a semantic config so users will need to know to go and create them

**Review by dereklegenzoff — COMMENTED** [review]
sorry, i didn't finish this review earlier

File: 02 - Web UI Template/CognitiveSearch.UI/appsettings.json
Comment (ruoccofabrizio) [reply 02 - Web UI Template/CognitiveSearch.UI/appsettings.json]: good catch!

**Comment by ruoccofabrizio** [comment]
@dereklegenzoff: thanks for your comments, just applied suggested mods. 

@farzad528: do you want to add anything on the README as Derek's suggestion or should I :)?

**Comment by farzad528** [comment]
Hey, go ahead and add whatever you’d like! I’ll revise next week (if needed).

Again, thanks for doing this!

Best,
 
Farzad Sunavala

**Comment by aldelar** [comment]
This would be great if added to this solution.
@liamca , @dereklegenzoff , @gmndrg would you be able to review / approve & merge if everything looks good?

**Comment by gmndrg** [comment]
@farzad528 are we good to proceed? I don't see any additions after the latest comment.

**Comment by gmndrg** [comment]
@aldelar there were some suggestions that were provided to @ruoccofabrizio and we are waiting for the ReadMe changes as oer the comments. As soon as applied, we'll be able to approve.

**Pull request merged**
